### PR TITLE
kill sequences

### LINKED ISSUE #1417: Remove Sequence (!)
Counter proposal to #1407:

Remove Sequence completely, and the Manifest has an `items` array of Canvases.  

* The Sequence behaviors are already available for Manifest, and this simplifies where they live.
* The multiple sequence case is not implemented in current viewers after 5+ years, and there are no known instances of Manifests with second or further Sequences
* It simplifies the default case of Manifest -> Sequence -> Canvas to just Manifest -> Canvas
* Ranges no longer have to be concerned with which Sequence they're applied to
* Ranges are still available for reordering of the set of Canvases of the Manifest
* Can put 'non-sequential' on Manifest to solve 1407
* Can put 'hidden' on Canvases in the Manifest to have non sequential Canvases but are still described.
* Can use 'hidden' Canvases to have sub-canvases in the main bucket
* Gets most of the benefits of id map with losing the simple order use case

### PR DESCRIPTION
**Pull request by azaroth42**
Closes #1417

### CODE REVIEW

File: source/api/presentation/3.0/index.md [hunk 1]
```diff
@@ -543,18 +528,20 @@ The value _MUST_ be an array of strings, taken from the table below or a URI.
...
-| `repeat` | Valid on Collections, Manifests, and Sequences, that include Canvases with at least the `duration` dimension.  When the client reaches the end of the duration of the final Canvas in the resource, and the "auto-advance" `behavior` is also in effect, then the client _SHOULD_ return to the first Canvas in the resource with the "repeat" `behavior` and start playing again. If the "auto-advance" `behavior` is not in effect, then the client _SHOULD_ render a navigation control for the user to manually return to the first Canvas. |
+| `paged` | Valid on Manifests, and Ranges, which include Canvases that have at least `height` and `width` dimensions. Canvases included in resources with this `behavior` represent pages in a bound volume, and _SHOULD_ be presented in a page-turning interface if one is available.  The first canvas is a single view (the first recto) and thus the second canvas likely represents the back of the object in the first canvas. If this is not the case, see the "non-paged" `behavior`. |
```
Comment (mikeapp) [inline source/api/presentation/3.0/index.md]: "Valid on Manifests and Ranges," (no comma) ?

File: source/api/presentation/3.0/index.md [hunk 2]
```diff
@@ -543,18 +528,20 @@ The value _MUST_ be an array of strings, taken from the table below or a URI.
...
+| `repeat` | Valid on Collections and Manifests, that include Canvases with at least the `duration` dimension.  When the client reaches the end of the duration of the final Canvas in the resource, and the "auto-advance" `behavior` is also in effect, then the client _SHOULD_ return to the first Canvas in the resource with the "repeat" `behavior` and start playing again. If the "auto-advance" `behavior` is not in effect, then the client _SHOULD_ render a navigation control for the user to manually return to the first Canvas. |
+| `sequence` | Valid on Ranges, which are items of the `structures` property of a Manifest. Ranges with this `behavior` represent different orderings of the Canvases listed in the `items` property of the Manifest, and user interfaces that interact with this order _SHOULD_ use the order within the selected Range, rather than the default order of `items`. |
```
Comment (mikeapp) [inline source/api/presentation/3.0/index.md]: Do we need comma after "Ranges"?

File: source/api/presentation/3.0/index.md [hunk 3]
```diff
@@ -78,15 +78,15 @@ Note that in the following descriptions, "object" is used to refer to the object
...
+Collectively the use cases require a model in which one can characterize the object (via the _Manifest_ resource) and the individual views of the object (_Canvas_ resources). Each view may have images, audio, video and other content resources associated with it (_Content_ resources) to allow the view to be rendered to the user appropriately. An object may also have sections; for example, a book may have chapters of several pages, or a play might be divided into acts and scenes (_Range_ resources) and there may be groups of objects (_Collection_ resources).  These resource types, along with their properties, make up the IIIF Presentation API.
```
Comment (jpstroop) [inline source/api/presentation/3.0/index.md]: > Collectively the use cases require a model
Collectively, the use cases require a model

File: source/api/presentation/3.0/index.md [hunk 4]
```diff
@@ -715,12 +702,12 @@ The value _MUST_ be an array of JSON objects.  Each item _MUST_ have the `id` an
...
+A link from this Manifest or Range to a Canvas, or part of a Canvas, that is contained within it. The reference to part of a Canvas is handled in the same way that Ranges reference parts of Canvases, by adding a fragment to the end of the Canvas's URI specifing the spatial and/or temporal segment of interest.  When processing this relationship, a client _SHOULD_ advance to the specified Canvas, or specified segment of the Canvas, when beginning navigation through the Range.  This allows the client to begin with the first Canvas that contains interesting content rather than requiring the user to manually skip past uninteresting content.
```
Comment (jpstroop) [inline source/api/presentation/3.0/index.md]: > or part of a Canvas, that is contained within it.

or part of a Canvas that is contained within it.

Comment (jpstroop) [reply source/api/presentation/3.0/index.md]: OK, I get it. It's still awkward, but what about:

A link from this Manifest or Range, to a Canvas, or part of a Canvas, that is contained within it

Comment (jpstroop) [reply source/api/presentation/3.0/index.md]: image

**Pull request merged**